README.adoc

```diff
@@ -95,4 +98,4 @@ This library was initialially developed by
 https://www.nist.gov/services-resources/software/reeper[NIST], and then forked
 by https://www.ribose.com[Ribose Inc] for futher imporvements, pleast check the
 https://www.nist.gov/services-resources/software/reeper[NIST] for details
-licensing and futher usages.
+licensing gnd futher usages.
```

Comment:
Typo! :wink:

Reply:
Man, you got some amazing eyes, I will update that in the next push!

Reply:
Thanks to git diff for that! :+1: